substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -1880,6 +1881,28 @@ impl<T: Config> StakingInterface for Pallet<T> {
 		}
 	}
 
+	fn force_withdraw(stash: &Self::AccountId, amount: Self::Balance) -> DispatchResult {
+		let mut ledger = Self::ledger(Stash(stash.clone()))?;
+		ledger.active = ledger.active.saturating_sub(amount);
```

Comment:
May be you should look to withdraw first from `ledger.unlocking` chunks before trying to withdraw from active?